includes/class-sensei-frontend.php

```diff
@@ -145,9 +145,13 @@ public function enqueue_scripts() {
 			}
 
 			Sensei()->assets->register( 'sensei-stop-double-submission', 'js/stop-double-submission.js', [], true );
-			Sensei()->assets->register( 'sensei-disable-complete-lesson-button', 'js/frontend/course-video/disable-complete-lesson-button.js', [], true );
 			Sensei()->assets->register( Sensei()->token . '-user-dashboard', 'js/user-dashboard.js', [ 'jquery-ui-tabs' ], true );
 
+			// Course Video javascript.
+			wp_register_script( 'sensei-course-video-youtube-iframe-api', 'https://www.youtube.com/iframe_api', [], Sensei()->version, true );
```

Comment:
Now reading the code, I realize what you meant here about the sensei version. I don't think we need to add the Sensei version there. Maybe we could add a string with a static version? I don't think they will change `https://www.youtube.com/iframe_api` and keep the same URL. WDYT?

Reply:
Hm. I do not know what to use as a version. There is no version on their side (in the script or the documentation), but there is a changelog.
Basically, the best option is not to pass the version here (as we don't control it), but it causes failed to build.
I put `unversioned` for now.